### DISCUSSION TITLE
part-ii/2018-113/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#45] [Question comments]
Part II 2018 Principles of Quantum Mechanics: Paper 1, Section II, D 

https://questions.tripos.org/part-ii/2018-113/

**Comment by silkca** [comment]
The first part is a standard calculation demonstrated in lectures. For the second, rewriting $\Delta$ in terms of $A$ and $A^{\dagger}$ allows you to directly use the provided expression for first-order shift.
